js/src/mobx/hardwareStore.js

```diff
@@ -51,6 +52,21 @@ export default class HardwareStore {
     }, HW_SCAN_INTERVAL);
   }
 
+  scanTrezor () {
+    return this._api.parity
+      .trezor('get_devices')
+      .then((message) => {
+        this.pinMatrixRequest = JSON.parse(message).map((path) => {
+          return { path: path, manufacturer: 'Trezor' };
```

Comment:
Don't we need a name here?

Reply:
The manufacturer is used to say "Please enter the pin for your {manufacturer} device", which I thought was good enough. If you have multiple trezors plugged in you might need a name to distinguish though, but I didn't think it would be all that important. You could plug them in and unlock one at a time to distinguish. But open to other suggestions, we'd need to propagate the name from the backend though, right now only USB path is pushed out.